### PR TITLE
DOC-1774: Update autocompleter `trigger` option

### PR DESCRIPTION
**Pull request by MitchCTiny**
Related Ticket: DOC-1774

Description of Changes:
* Update name of `ch` single character trigger option to `trigger` option with multiple character (string) support

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by tiny-ben-tran — APPROVED** [review]

**Review by lnewson — DISMISSED** [review]
You can't just remove the option. You need to mention it's deprecated in the relevant version and the replacement. You also need to mention the new flag only works in newer versions as remember this is the docs for all of 6.x, not just 6.2.

**Comment by MitchCTiny** [comment]
> You can't just remove the option. You need to mention it's deprecated in the relevant version and the replacement. You also need to mention the new flag only works in newer versions as remember this is the docs for all of 6.x, not just 6.2.

Should I also mention that it is slated for remnov

File: modules/ROOT/pages/autocompleter.adoc [hunk 1]
```diff
@@ -23,7 +23,12 @@ The two arguments this method take are:
 [cols="1,2,1,4",options="header"]
 |===
 |Name |Value |Requirement |Description
-|ch |string (of one character) |Required |The character to trigger the autocompleter.
+|ch |`+string+` |Required |The character to trigger the autocompleter.
```
Comment (lnewson) [inline modules/ROOT/pages/autocompleter.adoc:26]: You haven't fixed this fully... it's still missing the part about trigger replaced this and should be used instead.

- [ ] Correct details

Comment (lnewson) [reply modules/ROOT/pages/autocompleter.adoc:26]: I see actually you have this in `trigger` actually... it'd make more sense to have it here, as this is the option we don't want people to use moving forward.

Comment (MitchCTiny) [reply modules/ROOT/pages/autocompleter.adoc:26]: It is here, this table cell follows three lines as the partial cannot be on a line with anything else

Comment (lnewson) [reply modules/ROOT/pages/autocompleter.adoc:26]: It's in the wrong spot as I said in my last message.

File: modules/ROOT/pages/autocompleter.adoc [hunk 2]
```diff
@@ -23,7 +23,12 @@ The two arguments this method take are:
 [cols="1,2,1,4",options="header"]
 |===
 |Name |Value |Requirement |Description
-|ch |string (of one character) |Required |The character to trigger the autocompleter.
+|ch |`+string+` (of one character) |Required |The character to trigger the autocompleter.
+include::partial$misc/admon-deprecated-6.2v.adoc[]
+It has been replaced with the `+trigger+` option, which should be used instead.
+|trigger |`+string+` |Required |The string to trigger the autocompleter.
+include::partial$misc/admon-requires-6.2v.adoc[]
+It replaces the deprecated `+ch+` option, and should be used instead of that option.
```
Comment (lnewson) [inline modules/ROOT/pages/autocompleter.adoc:31]:
```suggestion
include::partial$misc/admon-requires-6.2v.adoc[]
```
You really don't need to be repeating yourself twice here.

File: modules/ROOT/pages/autocompleter.adoc (hunk 1 above)
Comment (lnewson) [reply modules/ROOT/pages/autocompleter.adoc]: Ahh right, you did have something for both which I missed sorry. That said you really don't need both and we don't normally have it for both so you can remove the `trigger` message as I left a suggest for.

Comment (MitchCTiny) [reply modules/ROOT/pages/autocompleter.adoc]: If this current change doesn't fix what you are asking then I'm sorry I just don't understand. The first line of this option (and of the `trigger` option) describes its use, then the line after it has the deprecation (or requires) partial since that cannot go on the same line as anything else, and then the third line states the correct option to use. I put the final part after the partial since it makes more sense that way, as it is stating the replacement immediately after the deprecation (or requires) notice.

File: modules/ROOT/pages/autocompleter.adoc (hunk 2 above)
Comment (MitchCTiny) [reply modules/ROOT/pages/autocompleter.adoc:31]: I thought that was what you were asking in the other thread?

File: modules/ROOT/pages/autocompleter.adoc [hunk 3]
```diff
@@ -23,7 +23,11 @@ The two arguments this method take are:
 [cols="1,2,1,4",options="header"]
 |===
 |Name |Value |Requirement |Description
-|ch |string (of one character) |Required |The character to trigger the autocompleter.
+|ch |`+string+` (of one character) |Required |The character to trigger the autocompleter.
+include::partial$misc/admon-deprecated-6.2v.adoc[]
+It has been replaced with the `+trigger+` option, which should be used instead.
```
Comment (ephox-mogran) [inline modules/ROOT/pages/autocompleter.adoc:28]: Do we explain why we deprecate it anywhere? I assume in this case it's because autocompleters can now have triggers that are more than one character, but do we need to say that? Do we need to provide the "no-op" migration advice? Or is that just something we cover in Release Notes?

**Review by ephox-mogran — APPROVED** [review]

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/autocompleter.adoc (hunk 3 above)
Comment (lnewson) [reply modules/ROOT/pages/autocompleter.adoc:28]: It's something that would normally go in the release notes if necessary from what I recall, as it ends up being too wordy to put here 🤷

Comment (MitchCTiny) [reply modules/ROOT/pages/autocompleter.adoc:28]: I can confirm that this is in the in progress release notes @brian-forte
